spyder/utils/introspection/test/test_fallback_plugin.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+#
+
+"""
+Tests for fallback_plugin.py
+"""
+
+# Standard library imports
+import os
+import os.path as osp
+
+# Test library imports
+import pytest
+
+# Local imports
+from spyder.utils.introspection.fallback_plugin import (FallbackPlugin,
+                                                        python_like_exts,
+                                                        all_editable_exts,
+                                                        get_parent_until,
+                                                        python_like_mod_finder
+                                                        )
+from spyder.utils.introspection.manager import CodeInfo
+
+FALLBACK_PLUGIN_FILE = osp.join(os.path.dirname(__file__), '..',
+                                'fallback_plugin.py')
+
+def test_fallback_plugin():
```

Comment:
Please break this test in several tests, instead a leaving it as a single, big one.